scl/cisco/plugin.conf

```diff
@@ -80,6 +80,7 @@ block parser cisco-triplet-parser(template() prefix()) {
             };
             rewrite { set("${`prefix`facility}-$1" value('`prefix`facility')); };
         };
+	rewrite { set-level("${`prefix`severity}"); };
     };
 };
```

Comment:
Ah, i get this now. Does this value also match the value in brackets at the front? Eg the <PRI> header?